examples/api/lib/material/button_style_button/button_style_button.icon_alignment.0.dart

```diff
@@ -0,0 +1,147 @@
+                    label: const Text('FilledButton'),
+                    iconAlignment: _iconAlignment,
+                  ),
+                  FilledButton.tonalIcon(
+                    onPressed: () {},
+                    icon: const Icon(Icons.cloud),
+                    label: const Text('FilledButton Tonal'),
+                    iconAlignment: _iconAlignment,
+                  ),
+                  OutlinedButton.icon(
+                    onPressed: () {},
+                    icon: const Icon(Icons.light),
+                    label: const Text('OutlinedButton'),
+                    iconAlignment: _iconAlignment,
+                  ),
+                  TextButton.icon(
+                    onPressed: () {},
+                    icon: const Icon(Icons.flight_takeoff),
+                    label: const Text('TextButton'),
+                    iconAlignment: _iconAlignment,
+                  ),
+                ],
+              ),
+              const Spacer(),
+              OverflowBar(
+                alignment: MainAxisAlignment.spaceEvenly,
+                overflowAlignment: OverflowBarAlignment.center,
+                spacing: 10,
+                overflowSpacing: 10,
+                children: <Widget>[
+                  Column(
+                    children: <Widget>[
+                      const Text('Icon alignment'),
+                      const SizedBox(height: 10),
+                      SegmentedButton<IconAlignment>(
+                        onSelectionChanged: (Set<IconAlignment> value) {
+                          setState(() {
+                            _iconAlignment = value.first;
+                          });
+                        },
+                        selected: <IconAlignment>{ _iconAlignment },
+                        segments: IconAlignment.values.map((IconAlignment iconAlignment) {
+                          return ButtonSegment<IconAlignment>(
+                            value: iconAlignment,
+                            label: Text(iconAlignment.name),
+                          );
+                        }).toList(),
+                      ),
+                    ],
+                  ),
+                  Column(
+                    children: <Widget>[
+                      const Text('Text direction'),
+                      const SizedBox(height: 10),
+                      SegmentedButton<TextDirection>(
+                        onSelectionChanged: (Set<TextDirection> value) {
+                          setState(() {
+                            _textDirection = value.first;
+                          });
+                        },
+                        selected: <TextDirection>{ _textDirection },
+                        segments: const <ButtonSegment<TextDirection>>[
+                          ButtonSegment<TextDirection>(
+                            value: TextDirection.ltr,
+                            label: Text('RTL'),
```

Comment:
```suggestion
                            value: TextDirection.ltr,
                            label: Text('LTR'),
```

Reply:
OMG! how I couldn't figure out this, sorry for the trouble. thank you!